debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/history/JdbcSchemaHistoryConfig.java

```diff
@@ -22,7 +22,7 @@
 public class JdbcSchemaHistoryConfig extends JdbcCommonConfig {
 
     private static final String DEFAULT_TABLE_NAME = "debezium_database_history";
-    private static final Field PROP_TABLE_NAME = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "schema.history.table.name")
+    public static final Field PROP_TABLE_NAME = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "schema.history.table.name")
```

Comment:
do we need to make it public?

Reply:
Hi @vjuranek , when we use debezium engine and if we need to get the value from properties, it might be better to use the field, the other ones are `public` for the same reason.

Reply:
I'd make it public when we need make it public

Reply:
ops, sorry, I missed this one